srcpkgs/eiwd/files/iwd/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+[ -r ./conf ] && . ./conf
+exec 2>&1
```

Comment:
This line should be removed as per #21353.

Reply:
add a ./log/run symlink to vlogger instead